piker/brokers/kraken.py

```diff
@@ -685,16 +715,31 @@ async def subscribe(ws: wsproto.WSConnection, token: str):
             trio.open_nursery() as n,
         ):
             ## TODO: maybe add multiple accounts
-            n.start_soon(handle_order_requests, client, ems_stream)
+            #n.start_soon(handle_order_requests, client, ems_stream)
```

Comment:
making sure you've noticed this.

as it stands if you don't relay the order request of course you can't expect to get status updates 😂

Reply:
why do I feel like this is old code?

Reply:
because it is 😂 ?

look at the timestamps of the comments lol